docs/contributors/build_guide.md

```diff
@@ -3,6 +3,15 @@
 * If you're not on Linux, you'll need a Docker installation
 * Minikube requires at least 4GB of RAM to compile, which can be problematic when using docker-machine
 
+#### Prerequisites for different GNU/Linux distributions
+
+##### Fedora
+On Fedora you need to install development-tools and glibc-static
+
+```shell
+$ sudo dnf install -y @development-tools glibc-static
```

Comment:
`development-tools` contain lot of packages check with `dnf groupinfo development-tools` we should be more explicit what package actually require for this purpose.